website/src/content/development/notifications.mdx

```diff
@@ -0,0 +1,387 @@
+---
+title: Notifications
+---
+
+In the previous sections, we learned how to develop forms. In this section we look at how to communicate feedback to users using notifications, in particular in response to user actions.
+
+To facilitate the use of notifications, Custom Applications come with a built-in notification system.
+
+# Notification system
```

Comment:
We have sections for `page` and `side` notifications but not for `global` ones.
Perhaps it would also be helpful to explain what use cases we recommend to use this kind of notifications for and how the look like.

Reply:
Thanks, we're omitting that on purpose (without going into too much detail) as this is mainly for internal use. We also don't document all the action creators/hooks that we use internally

Reply:
Ok. Thanks for the explanation 👍